commit 376eb2d39f54
Author: Doug Thayer <dothayer@mozilla.com>
Date:   Wed Sep 20 15:58:25 2017 -0700

    Bug 1399177 - Pull label out of description in update popups. r=Gijs, a=sledru
    
    The margin rules for labels mean that if the See what's new link
    starts its own line, the alignment looks off. We could override this,
    but matching the styling for "Learn more" links, which get their own
    line, seemed the better option.
    
    MozReview-Commit-ID: 4WK9QtRMUQs
    
    --HG--
    extra : source : f81f645704e4cdfaeebeb21d53d5d749718c3959
---
 browser/components/customizableui/content/panelUI.inc.xul | 10 ++++------
 1 file changed, 4 insertions(+), 6 deletions(-)

diff --git browser/components/customizableui/content/panelUI.inc.xul browser/components/customizableui/content/panelUI.inc.xul
index 86fc1fa5639f..e7c63f612dc6 100644
--- browser/components/customizableui/content/panelUI.inc.xul
+++ browser/components/customizableui/content/panelUI.inc.xul
@@ -421,9 +421,8 @@
                      buttonhighlight="true"
                      hidden="true">
     <popupnotificationcontent id="update-available-notification-content" orient="vertical">
-      <description id="update-available-description">&updateAvailable.message;
-        <label id="update-available-whats-new" class="text-link" value="&updateAvailable.whatsnew.label;" />
-      </description>
+      <description id="update-available-description">&updateAvailable.message;</description>
+      <label id="update-available-whats-new" class="text-link" value="&updateAvailable.whatsnew.label;" />
     </popupnotificationcontent>
   </popupnotification>
 
@@ -440,9 +439,8 @@
                      buttonhighlight="true"
                      hidden="true">
     <popupnotificationcontent id="update-manual-notification-content" orient="vertical">
-      <description id="update-manual-description">&updateManual.message;
-        <label id="update-manual-whats-new" class="text-link" value="&updateManual.whatsnew.label;" />
-      </description>
+      <description id="update-manual-description">&updateManual.message;</description>
+      <label id="update-manual-whats-new" class="text-link" value="&updateManual.whatsnew.label;" />
     </popupnotificationcontent>
   </popupnotification>
 
